evi-typescript-example/.env.example

```diff
@@ -1,2 +1,3 @@
 VITE_HUME_API_KEY="YOUR_API_KEY"
-VITE_HUME_SECRET_KEY="YOUR_SECRET_KEY"
\ No newline at end of file
+VITE_HUME_SECRET_KEY="YOUR_SECRET_KEY"
+VITE_HUME_CONFIG_ID="YOUR_CONFIG_ID"
```

Comment:
Notably, if the CONFIG_ID is left as its default value, "YOUR_CONFIG_ID" and the rest of the values are changed to their corresponding keys, then the WebSocket connection understandably errors out because "YOUR_CONFIG_ID" is not a valid config id.

Changing the value from "YOUR_CONFIG_ID" to and empty string fixes this issue, however it may be the case that we want to guide developers to using a config id

In any case, the README should be updated to mention the "YOUR_CONFIG_ID" field, near line 42 where the environment variables are discussed

Reply:
@norman-codes This is not the actual env file that gets used, the user needs to mae a .env file and model it after the .env.example file and fill in their own values.

Great callout on the readme! I'll update that there 🙏